documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,141 @@
+# WCA Competition Requirements Policy
+<div class="version">
+### Version 1.0
+</div>
+
+## Purpose
+The purpose of this policy is to define the requirements and processes that a competition must meet and follow in order to be recognized as a WCA Competition.
+
+## Policy
+### WCA Competitions
+An official WCA Competition must:
+
+1. Be announced at least 28 days (four weeks) before 00:00 UTC of the start date of the competition on the WCA website.
+   1. WCA Delegates should allow a suitable time frame for competitions to be announced after they are confirmed to the WCA Competition Announcement Team.
+2. Comply with the [WCA Regulations](wca{regulations}).
+3. Comply with all [WCA Policies](wca{documents}).
+4. Be publicly accessible.
+5. Be open to all competitors and spectators who wish to attend, up to the maximum capacity stated on the website or otherwise permitted by the venue.
+6. Be held on a single date or on two or more consecutive days.
+7. Have at least one official event.
+8. Have at least one competitor.
+9. Have at least one designated WCA Delegate in attendance.
```

Comment:
Apologies if this was already covered: should we remove "in attendance" as this is not true for 0.1% of competitions (estimated) and I would think that the board would wish to keep this exception for the 0.1% that have an emergency.

Reply:
Perhaps it makes sense to add this (stolen from Motion 8.2019.1):
Exception: in case the WCA Delegate cannot attend the competition, the WCA Board may
appoint a trusted person to act as a temporary WCA Candidate Delegate for the
competition.

Reply:
Sounds good to me!

```suggestion
9. Have at least one designated WCA Delegate in attendance.
   1. Exception: in case the WCA Delegate cannot attend the competition, the WCA Board may
appoint a trusted person to act as a temporary WCA Candidate Delegate for the
competition.
```